quickwit/quickwit-indexing/src/models/merge_scratch.rs

```diff
@@ -17,14 +17,14 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-use tantivy::Directory;
+use tantivy::{Directory, TrackedObject};
 
 use crate::merge_policy::MergeOperation;
 use crate::models::ScratchDirectory;
 
 #[derive(Debug)]
 pub struct MergeScratch {
-    pub merge_operation: MergeOperation,
+    pub merge_operation: TrackedObject<MergeOperation>,
```

Comment:
It is worth putting a minor comment, or even a link to a markdown explaining what this is about.